### DISCUSSION TITLE
Reanimated not working on Web

### DISCUSSION THREAD

**Discussion by moyolvera** [expo/expo#16322] [Archived: Questions]
## EDIT: 

The issue is solved if doing `yarn add react-native-reanimated` instead of `expo install react-native-reanimated`
I have a warning saying that that version is not supported:

`- react-native-reanimated - expected version: ~2.3.1 - actual version installed: 2.4.1`

But its working now, should this be labeled as an issue?

## end of edit

Hello!
I been having troubles to make Reanimated work in web, works fine in android and ios, i'm using the managed workflow.

Expo SDK: 44
react-native-reanimated: 2.3.1
react-native-web: 0.17.1

For the installation i followed the steps described in [expo documentation](https://docs.expo.dev/versions/latest/sdk/reanimated/)

When trying to run on web i can see that webpack bundles just fine with the message: `Web Bundling complete 4841ms`

But then it fails with the error:

`Module not found: Can't resolve 'react-native-reanimated'`

And in the browser console i see:

`Uncaught Error: Cannot find module 'react-native-reanimated' at webpackMissingModule`

As i said on mobile it works fine, the animations are working great but for web it's failing just after opening the page

this is my babel.config.js file

```
module.exports = function (api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    plugins: [
      [
        'module-resolver',
        {
          root: ['./src'],
          alias: {
            '@components': './src/components',
            '@modals': './src/modals',
            '@screens': './src/screens',
            '@utils': './src/utils',
            '@hooks': './src/hooks',
            '@actions': './src/actions'
          },
          extensions: ['.ts', '.tsx', 'web.ts', '.web.tsx']
        }
      ],
      'react-native-reanimated/plugin'
    ]
  };
};
```

I will greatly appreciate if you can refer to some web repos using reanimated that i can look on
Thanks!

**Comment by martinezguillaume** [comment]
I have exactly the same problem, not working on `~2.3.1` with this error `Module not found: Can't resolve 'react-native-reanimated'`.
Working fine with `^2.4.1`.

I tried to customize the webpack config file like so without success (from [reanimated doc](https://docs.swmansion.com/react-native-reanimated/docs/2.0.0/web-support/#webpack-support))

```tsx
// webpack.config.js
const createExpoWebpackConfigAsync = require('@expo/webpack-config');
const webpack = require('webpack');

module.exports = async function (env, argv) {
  const config = await createExpoWebpackConfigAsync(env, argv);

  config.entry.app.unshift('babel-polyfill')
  config.plugins.push(new webpack.DefinePlugin({
    process: { env: {} }
  }))
  config.resolve.extensions.push('.web.js', '.js')

  return config;
};
```
